src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -58,7 +59,7 @@ private void writeToFile(String toFileName, String[] array) {
         }
     }
 
-    private String[] readFromFile(String fileName) {
+    private String readFromFile(String fileName) {
         StringBuilder stringBuilder = new StringBuilder();
         try {
             BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
```

Comment:
BufferedReader must be closed at the end of reading. You can use 'try with resources' for it. Or you can use block 'finally'